libctru/include/3ds/services/frd.h

```diff
@@ -93,9 +93,9 @@ Result FRD_GetMyProfile(Profile *profile);
 
 /**
  * @brief Gets the current user's screen name.
- * @param name Pointer to write the current user's screen name to. 11-byte UTF-16 screen name (with null terminator)
+ * @param name Pointer to write the current user's screen name to.
  */
-Result FRD_GetMyScreenName(u16 *name);
+Result FRD_GetMyScreenName(u8 *name);
```

Comment:
Should be char*, and should have a parameter to describe the output buffer size (and use it).